dashboard/app/models/pd/workshop.rb

```diff
@@ -334,6 +334,12 @@ def friendly_name
     "#{course_subject} workshop on #{start_time} at #{location_name}"[0...255]
   end
 
+  def short_name
+    return '' unless sessions.size && processed_location
```

Comment:
I don't think you want `sessions.size` here. In Ruby, 0 is truthy:
```
irb(main):006:0> !!0
=> true
irb(main):007:0> !0
=> false
```

Maybe `sessions.any?` ?

Reply:
Also this seems kind of specific to the teachercon / summer workshop view. I'm not sure that `short_name` is entirely appropriate. We already have a `friendly_name` and this isn't necessarily shorter. I think this might better belong in the serializer.

Or if we want to leave it in the model, at least name it more appropriately: something like `friendly_date_and_location_name`.

Reply:
I'm fine moving this to the serializer